main.go

```diff
@@ -0,0 +1,139 @@
+package main
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"sort"
+)
+
+type Record struct {
+	ID        uint
+	FirstName string
+	LastName  string
+	PhoneN    string
+}
+
+func removeEntry() (errr string) {
+	var Records []Record
+	f, err := os.Open("Data.json")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer f.Close()
+	if err := json.NewDecoder(bufio.NewReader(f)).Decode(&Records); err != nil {
+		log.Fatal(err)
+	}
+	id := -1
+	fmt.Println("ID:")
+	fmt.Scanln(&id)
+	for i, v := range Records {
+		if v.ID == uint(id) {
+			Records = append(Records[:i], Records[i+1:]...)
+			f.Close()
+			f, _ = os.Create("Data.json")
+			defer f.Close()
+			w := bufio.NewWriter(f)
+			defer w.Flush()
+			if err := json.NewEncoder(w).Encode(Records); err != nil {
+				log.Fatal(err)
+			}
+			return ""
+		}
+	}
+	return "ID not found"
+}
+
+func getNextID(rec []Record) uint {
+	id := -1
+	for _, v := range rec {
+		if id < int(v.ID) {
+			id = int(v.ID)
+		}
+	}
+	return uint(id + 1)
+}
+
+func ListEntries() {
+	var Records []Record
+	f, err := os.Open("Data.json")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer f.Close()
+	if err := json.NewDecoder(bufio.NewReader(f)).Decode(&Records); err != nil {
+		log.Fatal(err)
+	}
+	sort.Slice(Records, func(i, j int) bool { return Records[i].LastName < Records[j].LastName })
+	fmt.Println("ID              Last Name                First Name                Phone#")
+	fmt.Println("-------------------------------------------------------------------------")
+	i := 0
+	for _, rec := range Records {
+		if i == 20 {
```

Comment:
What if there are more than 20 records? 1234 for instance. Your code would show the pagination only once.